hanyu/templatetags/AdminLTE.py

```diff
@@ -8,24 +8,41 @@
 register = template.Library()
 
 
-@register.inclusion_tag('_menu_tree.html', takes_context=True)
-def menu_tree(context, heading, *args):
+@register.inclusion_tag('_menu_tree_open.html', takes_context=True)
+def menu_tree_open(context, heading):
     '''
-    This is a tag used to build the AdminLTE 2 tree menu
-    The first argument is the menu heading.
-    The following arguments are alternating namespace:urls and the description
+    heading         := Menu heading
+    descriptions    := list of sub-menu link descriptions
+    urls            := list of sub-menu urls
+    args            := list of tuples of keyword, argument pairs [((k,a),..., (k,a)),..., ((k,a),...) ]
     '''
-    links = []
-    for url, desc in zip(args[0::2], args[1::2]):
-        links.append({'url':reverse(url), 'desc':desc})
-
     context['heading'] = heading
-    context['links'] = links
+    return context
+
+
+@register.inclusion_tag('_menu_tree_item.html', takes_context=True)
+def menu_tree_item(context, url_namespace, description, *args):
+    '''
+    description    := list of sub-menu link descriptions
```

Comment:
I'm not sure where this docstring syntax comes from, but if you want to use sphynx you should use the ReStructured Text format:

http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html